config/gov_notify.yml

```diff
@@ -30,6 +34,8 @@ development:
 
 test:
   support_email: 'test@localhost'
+  otp_email:
+    template_id: '776fff48-3e94-4417-83b0-bff027c16247'
```

Comment:
Should be create one for production too? Might be good to set it up now rather than a separate PR later but let me know if you feel otherwise.

Reply:
ok!